src/containers/counter.js

```diff
@@ -0,0 +1,85 @@
+// @ts-check
+
+import React, { Component } from "react";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import { StyleSheet, Text, View, FlatList } from "react-native";
+import { Button } from "react-native-elements";
+import {
+  counterIncrement,
+  counterDecrement,
+  newCounter,
+  closeCounter
+} from "../actions/counter";
+import CounterRow from "../components/counter";
+import uuid from "uuid";
+
+const styles = StyleSheet.create({
+  counter_row: {
+    padding: 5,
+    flexDirection: "row",
+    justifyContent: "center",
+    alignItems: "stretch"
+  },
+  counter_col: {
+    flex: 1,
+    flexDirection: "column"
+  }
+});
+
+class CounterList extends Component {
+  renderItem({ data, index }) {
+    return (
+      <CounterRow
+        index={index}
+        data={this.props.counterList}
```

Comment:
Why not just pass the cell data directly instead of the whole list? Looking at the CounterRow component and the actions it fires it only seems to need to know about the cell data itself. It should also clean up the CounterRow code by no longer needing lines like this to get the count: `this.props.data[this.props.index].count`.